Source/monster.cpp

```diff
@@ -1347,28 +1331,35 @@ void MonsterAttackPlayer(int monsterId, int pnum, int hit, int minDam, int maxDa
 			player.position.tile = newPosition;
 			FixPlayerLocation(player, player._pdir);
 			FixPlrWalkTags(player);
-			dPlayer[newPosition.x][newPosition.y] = pnum + 1;
+			dPlayer[newPosition.x][newPosition.y] = player.getId() + 1;
 			SetPlayerOld(player);
 		}
 	}
 }
 
-bool MonsterAttack(int monsterId)
+void MonsterAttackEnemy(Monster &monster, int hit, int minDam, int maxDam)

```

Comment:
Preferably this should be in an anonymous namespace since it is only used in this file.

Reply:
I think that it already is anonymous

<img width="917" alt="image" src="https://user-images.githubusercontent.com/288227/180666966-ad75400f-f570-43a0-b382-ae294dd97365.png">